CONTRIBUTING.md

```diff
@@ -3,74 +3,98 @@
 Contributing encompasses repository specific requirements.
 
 ## Install
+
 Before developing you'll need to install:
-* [NodeJS and NPM](https://nodejs.org/)
-* [OpenShift CLI](https://docs.openshift.com/enterprise/3.2/cli_reference/get_started_cli.html#installing-the-cli)
-* [kustomize](https://github.com/kubernetes-sigs/kustomize)
-* [podman](https://github.com/containers/podman)
-* and have access to [Quay.io](https://quay.io/)
+
+- [NodeJS and NPM](https://nodejs.org/)
+  - Node recommended version -> `18.14.0`
+  - NPM recommended version -> `8.19.4`
+- [OpenShift CLI](https://docs.openshift.com/container-platform/4.12/cli_reference/openshift_cli/getting-started-cli.html)
+- [kustomize](https://github.com/kubernetes-sigs/kustomize)
+- [podman](https://github.com/containers/podman)
+- and have access to [Quay.io](https://quay.io/)
 
 ## Writing code
+
 ### Running locally
+
 Development for both "frontend" and "backend" can be done while running
-```
-$ npm run dev
+
+``` bash
+npm run dev
```

Comment:
This is good in a pinch -- but it sucks for actual development. You can't stop the server and client independently... so if you want to clear a cache or need to restart the frontend or backend independently you have to drop everything.

Did we want to note you can start them via `npm run start:dev` in the frontend and backend folders?